src/style.css

```diff
@@ -1,3 +1,68 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__link {
+  display: inline-block;
+  margin: 0 10px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
```

Comment:
color '#00acdc' is duplicated, so please move all duplicated styles to the variables